### DISCUSSION TITLE
Garage node

### DISCUSSION THREAD

**Discussion by olapola85** [andrei-tatar/node-red-contrib-smartnora#173] [Q&A] [answered]
In the google home app on iOS. I cant press the garage icon to open. I have to speak to the phone to open/close the garage door

Is this the way its supposed to be?

[image](https://user-images.githubusercontent.com/35819757/162724886-eb62e6f7-b95a-45af-b220-67e4fb9f905b.jpeg)

**Comment by rgerrans** [comment, marked as answer]
Correct. Google control is voice-oriented so many of the phone (and a few less of the Hub) icons are non-functioning. Lock is an example that you can control on a Hub but not on a phone

**Reply by andrei-tatar** [reply]
@olapola85 to give a bit more details, here's the list of device types/traits you can control depending on the device you are using (smart display/assistant app/google home app):
https://developers.google.com/assistant/smarthome/develop/touch-controls

**Reply by olapola85** [reply]
Thanks everyone! So I cant view the status of the door in the app either? I have to ask google?

**Reply by andrei-tatar** [reply]
@olapola85 Yes
